orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -180,17 +184,17 @@ def SendViaFTPS(resourceId: str) -> None:
 
 def ShouldAutoRoute():
     """
-    Checks whether ORTHANC_AUTOROUTE_ANON_TO_AZURE environment variable is
+    Checks whether ORTHANC_AUTOROUTE_ANON_TO_FTPS environment variable is
     set to true or false
     """
-    return os.environ.get("ORTHANC_AUTOROUTE_ANON_TO_AZURE", "false").lower() == "true"
+    return os.environ.get("ORTHANC_AUTOROUTE_ANON_TO_FTPS", "false").lower() == "true"
 
 
-def OnChange(changeType, level, resource) -> None:  # noqa: ARG001
+def OnChange(changeType: str, _level: None, resource: str) -> Any:
```

Comment:
I'm guessing there's a reason with the orthanc plugin system that means we've changed these to returning `Any`?

Reply:
Tbf, I just added these to get rid of the overload of squiggly lines in my VSCode 😅 
There weren't any type annotations before and we're not enforcing them in this file.
(I guess I could also just ignore that file in my VSCode...)